asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/HttpRequestFactory.java

```diff
@@ -147,6 +162,20 @@ HttpPost updateAttachmentContentRequest(String contentId, String attachmentId, I
         return attachmentPostRequest;
     }
 
+    HttpPost updateAttachmentContentRequest(String contentId, String attachmentId, Path attachmentContent, boolean notifyWatchers) {
+        assertMandatoryParameter(isNotBlank(contentId), "contentId");
+        assertMandatoryParameter(isNotBlank(attachmentId), "attachmentId");
+        assertMandatoryParameter(attachmentContent != null, "attachmentContent");
+
+        HttpPost attachmentPostRequest = new HttpPost(this.confluenceRestApiEndpoint + "/content/" + contentId + "/child/attachment/" + attachmentId + "/data");
+        attachmentPostRequest.addHeader(new BasicHeader("X-Atlassian-Token", "no-check"));
+
+        HttpEntity multipartEntity = multipartEntity(null, attachmentContent, notifyWatchers);
+        attachmentPostRequest.setEntity(multipartEntity);
+
+        return attachmentPostRequest;
+    }
```

Comment:
Looks like there is quite some overlap between these 2 new methods:
- `addAttachmentRequest()`, and
- `updateAttachmentContentRequest()`

Would it make sense to share the code, e.g. by introducing a delegate method with 1 or 2 additional parameters? Probably ensure 100% unit test coverage first, and then consider a refactoring?